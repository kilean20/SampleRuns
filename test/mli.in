#menu 

beam: beam, ekinetic=0.0025, mass=0.938272045, charge=1.0 
units: units, type=static 
setdefaults: globaldefaults, lfrngsbend=1, tfrngsbend=1, lfrngquad=0, tfrngquad=0, driftexact=1 
dipole1: sbend, l=0.2, angle=0.5235987755982988, hgap=0.015, fint=0.5 
ptm: ptm, matrix=3, poly=3 

impactLattice, line=( dipole1 ) 

#labor 
impactLattice 
ptm 
