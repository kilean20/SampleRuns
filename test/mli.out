 sbend, user input l=  0.20000000000000001      code computed rho=  0.38197186342054890     
 sbend, B=  0.59853067526923676     

 matrix for map is :

  8.66025E-01  1.90986E-01  0.00000E+00  0.00000E+00  0.00000E+00 -7.02425E-01
 -1.30900E+00  8.66025E-01  0.00000E+00  0.00000E+00  0.00000E+00 -6.86303E+00
  0.00000E+00  0.00000E+00  1.02057E+00  2.00000E-01  0.00000E+00  0.00000E+00
  0.00000E+00  0.00000E+00  2.07839E-01  1.02057E+00  0.00000E+00  0.00000E+00
  6.86303E+00  7.02425E-01  0.00000E+00  0.00000E+00  1.00000E+00  3.57827E+01
  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  0.00000E+00  1.00000E+00

 nonzero matrix elements in full precision:
1  1   0.86602540378444    
1  2   0.19098593171027    
1  6  -0.70242465111855    
2  1   -1.3089969389957    
2  2   0.86602540378444    
2  6   -6.8630303368106    
3  3    1.0205722519223    
3  4   0.20000000000000    
4  3   0.20783860696920    
4  4    1.0205722519223    
5  1    6.8630303368106    
5  2   0.70242465111855    
5  5    1.0000000000000    
5  6    35.782655116152    
6  6    1.0000000000000    


 nonzero elements in generating polynomial are :

  f( 28)=f( 30 00 00 )=-0.14278941552502    
  f( 29)=f( 21 00 00 )=-0.28340615066160    
  f( 33)=f( 20 00 01 )= -2.2459214257800    
  f( 34)=f( 12 00 00 )=-0.18750000000000    
  f( 38)=f( 11 00 01 )= -2.9717793093106    
  f( 39)=f( 10 20 00 )= -1.7873432636929    
  f( 40)=f( 10 11 00 )= 0.75091982350047    
  f( 43)=f( 10 02 00 )=-0.31853891945201    
  f( 48)=f( 10 00 02 )= -58.626481754978    
  f( 49)=f( 03 00 00 )= 2.23123100801237E-02
  f( 53)=f( 02 00 01 )=-0.98305668242373    
  f( 54)=f( 01 20 00 )= 0.12852086917065    
  f( 55)=f( 01 11 00 )= 0.45751444920134    
  f( 58)=f( 01 02 00 )=-1.97577210221187E-02
  f( 63)=f( 01 00 02 )= -2.9952989296839    
  f( 67)=f( 00 20 01 )= -9.3709852755110    
  f( 70)=f( 00 11 01 )=  3.9370493357685    
  f( 76)=f( 00 02 01 )= -1.6700896713564    
  f( 83)=f( 00 00 03 )= -266.21848255634    
  f( 89)=f( 30 00 01 )= -1.9599361810473    
  f( 90)=f( 22 00 00 )=-4.09061543436171E-02
  f( 94)=f( 21 00 01 )= -3.8900500192585    
  f( 95)=f( 20 20 00 )=-0.37073709125979    
  f( 96)=f( 20 11 00 )= 0.90040180820972    
  f( 99)=f( 20 02 00 )=-0.21225345297364    
  f(104)=f( 20 00 02 )= -38.493661043709    
  f(105)=f( 13 00 00 )=-5.41265877365274E-02
  f(109)=f( 12 00 01 )= -3.0025757723546    
  f(110)=f( 11 20 00 )= -1.4047996076547    
  f(111)=f( 11 11 00 )=  1.3439075391894    
  f(114)=f( 11 02 00 )=-0.14995181436623    
  f(119)=f( 11 00 02 )= -50.934402297525    
  f(123)=f( 10 20 01 )= -28.420708215212    
  f(126)=f( 10 11 01 )=  19.748728545613    
  f(132)=f( 10 02 01 )= -6.5979610719827    
  f(139)=f( 10 00 03 )= -885.09592233553    
  f(140)=f( 04 00 00 )=-1.79049310978382E-02
  f(144)=f( 03 00 01 )= 2.24761018154253E-02
  f(145)=f( 02 20 00 )=-0.88638160830170    
  f(146)=f( 02 11 00 )= 6.67169955481511E-02
  f(149)=f( 02 02 00 )=-3.67206168781574E-02
  f(154)=f( 02 00 02 )= -22.447381275033    
  f(158)=f( 01 20 01 )= -5.6012355080500    
  f(161)=f( 01 11 01 )=  13.325936619248    
  f(167)=f( 01 02 01 )= -1.0573883876783    
  f(174)=f( 01 00 03 )= -94.296125808480    
  f(175)=f( 00 40 00 )= -1.1716192861799    
  f(176)=f( 00 31 00 )= 0.52734702669789    
  f(179)=f( 00 22 00 )=-0.39423564279595    
  f(184)=f( 00 20 02 )= -201.51516690510    
  f(185)=f( 00 13 00 )= 7.79652751811643E-02
  f(190)=f( 00 11 02 )=  93.149805535962    
  f(195)=f( 00 04 00 )=-3.04182261587630E-02
  f(200)=f( 00 02 02 )= -38.932807133958    
  f(209)=f( 00 00 04 )= -4703.3008642186    
  f(221)=f( 30 20 00 )= 0.75523367531649    
  f(222)=f( 30 11 00 )=-0.29721628591128    
  f(225)=f( 30 02 00 )=-0.20905012048432    
  f(236)=f( 21 20 00 )= -1.2060801974896    
  f(237)=f( 21 11 00 )= 0.97833764547862    
  f(240)=f( 21 02 00 )=-0.18932438828528    
  f(249)=f( 20 20 01 )=  1.7133619808112    
  f(252)=f( 20 11 01 )=  19.812034097561    
  f(258)=f( 20 02 01 )= -7.9919767209710    
  f(271)=f( 12 20 00 )= -2.9834187951033    
  f(272)=f( 12 11 00 )=  1.1022688398581    
  f(275)=f( 12 02 00 )= 4.23917712952226E-03
  f(284)=f( 11 20 01 )= -51.195890042044    
  f(287)=f( 11 11 01 )=  46.846219722840    
  f(293)=f( 11 02 01 )= -4.6158495367616    
  f(301)=f( 10 40 00 )= -1.8160416530183    
  f(302)=f( 10 31 00 )=  3.0146648947641    
  f(305)=f( 10 22 00 )=-0.81103369905559    
  f(310)=f( 10 20 02 )= -106.06810987592    
  f(311)=f( 10 13 00 )= 0.32103408178323    
  f(316)=f( 10 11 02 )=  300.87279426984    
  f(321)=f( 10 04 00 )=-3.76520822488745E-02
  f(326)=f( 10 02 02 )= -74.740883424774    
  f(341)=f( 03 20 00 )=-0.58801863794602    
  f(342)=f( 03 11 00 )= 0.26689198810468    
  f(345)=f( 03 02 00 )=-5.19309041974412E-03
  f(354)=f( 02 20 01 )= -39.962892947540    
  f(357)=f( 02 11 01 )=  7.3747413381222    
  f(363)=f( 02 02 01 )= 0.16106650650374    
  f(371)=f( 01 40 00 )= -3.0029777817337    
  f(372)=f( 01 31 00 )=  2.4798344203427    
  f(375)=f( 01 22 00 )=-0.65669506642413    
  f(380)=f( 01 20 02 )= -330.39636258572    
  f(381)=f( 01 13 00 )= 0.27319066142833    
  f(386)=f( 01 11 02 )=  275.56572072579    
  f(391)=f( 01 04 00 )= 5.94219721586832E-03
  f(396)=f( 01 02 02 )= -18.568408017275    
  f(409)=f( 00 40 01 )= -41.684811491328    
  f(412)=f( 00 31 01 )=  30.279736069237    
  f(418)=f( 00 22 01 )= -15.033224541607    
  f(425)=f( 00 20 03 )= -2212.9404589484    
  f(428)=f( 00 13 01 )=  3.5538347288557    
  f(435)=f( 00 11 03 )=  1333.6677209894    
  f(443)=f( 00 04 01 )=-0.37708241305933    
  f(450)=f( 00 02 03 )= -235.97543051491    
  f(473)=f( 40 20 00 )=-0.27695599109502    
  f(474)=f( 40 11 00 )=-0.44422590917142    
  f(477)=f( 40 02 00 )= 0.14978667871807    
  f(488)=f( 31 20 00 )=-0.26121173002467    
  f(489)=f( 31 11 00 )= 0.91923569553949    
  f(492)=f( 31 02 00 )= 2.98686599974829E-02
  f(501)=f( 30 20 01 )=  9.7101827255105    
  f(504)=f( 30 11 01 )= -14.830859282410    
  f(510)=f( 30 02 01 )= -4.1027485916581    
  f(523)=f( 22 20 00 )= -1.7522883548080    
  f(524)=f( 22 11 00 )=  1.3088044995153    
  f(527)=f( 22 02 00 )=-0.17361444378172    
  f(536)=f( 21 20 01 )= -60.705784818281    
  f(539)=f( 21 11 01 )=  44.252726564868    
  f(545)=f( 21 02 01 )= -4.7714478653385    
  f(553)=f( 20 40 00 )= 0.66688953029792    
  f(554)=f( 20 31 00 )=  1.8721958345171    
  f(557)=f( 20 22 00 )= -1.5097849158214    
  f(562)=f( 20 20 02 )=  148.99322054649    
  f(563)=f( 20 13 00 )= 4.48935477306625E-02
  f(568)=f( 20 11 02 )=  20.567809959318    
  f(573)=f( 20 04 00 )=-8.59855901497485E-02
  f(578)=f( 20 02 02 )= -141.92609841364    
  f(593)=f( 13 20 00 )= -1.9234871916932    
  f(594)=f( 13 11 00 )= 0.30453032212873    
  f(597)=f( 13 02 00 )=-6.50137702808523E-02
  f(606)=f( 12 20 01 )= -111.30708898639    
  f(609)=f( 12 11 01 )=  44.016702011293    
  f(615)=f( 12 02 01 )= -2.2642515806192    
  f(623)=f( 11 40 00 )= -7.1632462921877    
  f(624)=f( 11 31 00 )=  6.2503056663428    
  f(627)=f( 11 22 00 )= -2.7723856980180    
  f(632)=f( 11 20 02 )= -1045.1098872577    
  f(633)=f( 11 13 00 )= 0.46397425448758    
  f(638)=f( 11 11 02 )=  717.70088018876    
  f(643)=f( 11 04 00 )=-3.45397591599690E-02
  f(648)=f( 11 02 02 )= -79.512169305601    
  f(661)=f( 10 40 01 )= -38.656463756658    
  f(664)=f( 10 31 01 )=  113.59832996464    
  f(670)=f( 10 22 01 )= -38.660467196121    
  f(677)=f( 10 20 03 )= -482.42341707459    
  f(680)=f( 10 13 01 )=  11.942023025470    
  f(687)=f( 10 11 03 )=  3471.7681380427    
  f(695)=f( 10 04 01 )= -2.5752302810142    
  f(702)=f( 10 02 03 )= -1400.2568161978    
  f(719)=f( 04 20 00 )=-0.52419686778859    
  f(720)=f( 04 11 00 )= 6.72026087241643E-02
  f(723)=f( 04 02 00 )= 1.29015063245288E-03
  f(732)=f( 03 20 01 )= -22.201497505874    
  f(735)=f( 03 11 01 )=  5.0375384314750    
  f(741)=f( 03 02 01 )=-0.71936531263813    
  f(749)=f( 02 40 00 )= -4.8744263559722    
  f(750)=f( 02 31 00 )=  2.9276779161739    
  f(753)=f( 02 22 00 )= -1.3392037057458    
  f(758)=f( 02 20 02 )= -812.28023301297    
  f(759)=f( 02 13 00 )= 7.84480037757840E-02
  f(764)=f( 02 11 02 )=  222.42817798608    
  f(769)=f( 02 04 00 )=-6.37032150872484E-03
  f(774)=f( 02 02 02 )= -6.9760175161476    
  f(787)=f( 01 40 01 )= -122.55543669885    
  f(790)=f( 01 31 01 )=  120.74709540883    
  f(796)=f( 01 22 01 )= -35.496203160073    
  f(803)=f( 01 20 03 )= -6033.2092643442    
  f(806)=f( 01 13 01 )=  6.9259827168638    
  f(813)=f( 01 11 03 )=  3539.4555911133    
  f(821)=f( 01 04 01 )=-0.30995079135022    
  f(828)=f( 01 02 03 )= -319.67755403444    
  f(840)=f( 00 60 00 )= -1.5579161455753    
  f(841)=f( 00 51 00 )=  2.3393597991532    
  f(844)=f( 00 42 00 )=-0.92101554551341    
  f(849)=f( 00 40 02 )= -534.36205146953    
  f(850)=f( 00 33 00 )= 0.57769431182256    
  f(855)=f( 00 31 02 )=  756.25119330790    
  f(860)=f( 00 24 00 )=-0.10973234168324    
  f(865)=f( 00 22 02 )= -262.72585999789    
  f(874)=f( 00 20 04 )= -17939.536307423    
  f(875)=f( 00 15 00 )= 3.30251121707394E-02
  f(880)=f( 00 13 02 )=  87.577381217876    
  f(889)=f( 00 11 04 )=  19537.489551276    
  f(896)=f( 00 06 00 )=-2.53399181958389E-03
  f(901)=f( 00 04 02 )= -13.469216110173    
  f(910)=f( 00 02 04 )= -4492.6423766088    
